crates/ethereum/node/src/launch.rs

```diff
@@ -159,22 +158,17 @@ where
 
         let pruner_events = pruner.events();
         info!(target: "reth::cli", prune_config=?ctx.prune_config().unwrap_or_default(), "Pruner initialized");
-        hooks.add(PruneHook::new(pruner, Box::new(ctx.task_executor().clone())));
```

Comment:
instead of in hooks, pruner is consumed when creating the service, otherwise we should be able to clone it or have difference pruners, wdyt?

Reply:
Making the service consume the pruner makes sense to me imo, since we will do things with the pruner in the eth service

Reply:
I like this